docs/tutorials/fundamentals/getting_started.md

```diff
@@ -34,10 +34,8 @@ This command will launch an empty viewer:
 ![image: an empty napari viewer](../assets/tutorials/launch_cli_empty.png)
 
 Once you have the viewer open you can add images through the `File -> Open` dropdown menu
-or by dragging and dropping images directly on the viewer.
-We currently only support files that can be read with [`skimage.io.imread`](https://scikit-image.org/docs/dev/api/skimage.io.html#skimage.io.imread),
-such as `tif`, `png`, and `jpg`.
-We plan on adding support for more exotic file types shortly - see [issue #379](https://github.com/napari/napari/issues/379) for discussion.
+or by dragging and dropping images directly on the viewer. We currently only support files that can be read with [`skimage.io.imread`](https://scikit-image.org/docs/dev/api/skimage.io.html#skimage.io.imread),
+such as `tif`, `png`, and `jpg`.  We plan on adding support for more exotic file types shortly - see [issue #379](https://github.com/napari/napari/issues/379) for discussion. Finally, you can use the `File -> New Image from Clipboard` menu item to make a new `Image` layer from an image (or URL to an image) in your Clipboard (keybinding {kbd}`Command/Ctrl+N`).
```

Comment:
Should we also make mention here of `File -> Open with plugin`?

Reply:
I think that's not related to the feature being documented here?
And frankly I don't know what that menu item does because normal File > Open File uses plugins to open...
Frankly, I think we need an `opening files` guide/tutorial whatever that documents what files can be opened by napari and that other files can be opened by plugins.

Reply:
>  And frankly I don't know what that menu item does because normal File > Open File uses plugins to open...

I think that difference is explained here: https://github.com/napari/napari/pull/4347
tl;dr - open tries to open with the preferred plugin from settings and open with plugin forces popping up of dialogue to choose plugin to open with. Practically, they both call `_open_files_dialog`, one just gives `choose_plugin=True` arg.

Reply:
Definitely think `File > Open with plugin` should get proper documentation. I added that to the existing issue:
https://github.com/napari/docs/issues/44
I think it's outside the scope of this PR which documents a specific feature recently added.

In general the opening/saving options of vanilla napari and key plugins needs dedicated docs.